src/custom_nodes/east_ocr/config.json

```diff
@@ -0,0 +1,102 @@
+{
+    "model_config_list": [
+        {
+            "config": {
+                "name": "east",
+                "base_path": "/models/east_icdar2015_resnet_v1_50_rbox"
+            }
+        },
+        {
+            "config": {
+                "name": "crnn",
+                "base_path": "/models/crnn_tf"
+            }
+        }
+    ],
+    "custom_node_library_config_list": [
+        {
+            "name": "east_image_extractor",
+            "base_path": "/libs/libcustom_node_east_ocr.so"
+        }
+    ],
+    "pipeline_config_list": [
+        {
+            "name": "detect_text_images",
+            "inputs": ["image"],
+            "nodes": [
+                {
+                    "name": "east_node",
+                    "model_name": "east",
+                    "type": "DL model",
+                    "inputs": [
+                        {"input_images": {"node_name": "request",
+                                          "data_item": "image"}}
+                    ],
+                    "outputs": [
+                        {"data_item": "feature_fusion/Conv_7/Sigmoid",
+                         "alias": "scores"},
+                        {"data_item": "feature_fusion/concat_3",
+                         "alias": "geometry"}
+                    ]
+                },
+                {
+                    "name": "extract_node",
+                    "library_name": "east_image_extractor",
+                    "type": "custom",
+                    "demultiply_count": 0,
+                    "params": {
+                        "original_image_width": "1920",
+                        "original_image_height": "1024",
+                        "target_image_width": "100",
+                        "target_image_height": "32",
+                        "convert_to_gray_scale": "false",
+                        "confidence_threshold": "1.0",
```

Comment:
DEbug 1.0? Normally should be something less?

Reply:
We should set it to 0.9